teachopencadd/talktorials/T014_binding_site_detection/talktorial.ipynb

```diff
@@ -0,0 +1,2116 @@
+{
```

Comment:
<p>What do you mean by "disease modifying target"? Maybe rather "a target (that causes a disease) to bind (and potentially be modulated by)"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Need to verify the actual definition - though there might exist several - the target does not necessarily need to 'cause' the disease. </p><p>Quickly check in <a href="https://en.wikipedia.org/wiki/Druggability" target="_blank">wikipedia</a> only :)</p><blockquote><strong>Druggability</strong>&nbsp;is a term used in&nbsp;<a href="https://en.wikipedia.org/wiki/Drug_discovery" target="_blank">drug discovery</a>&nbsp;to describe a&nbsp;<a href="https://en.wikipedia.org/wiki/Biological_target" target="_blank">biological target</a>&nbsp;(such as a&nbsp;<a href="https://en.wikipedia.org/wiki/Protein" target="_blank">protein</a>) that is known to or is predicted to bind with high affinity to a&nbsp;<a href="https://en.wikipedia.org/wiki/Drug" target="_blank">drug</a>. Furthermore, by definition, the binding of the drug to a druggable target must alter the function of the target with a therapeutic benefit to the patient.&nbsp;</blockquote>

Reply:
<p>Kept the statement, but added a reference with more details on druggablity.</p>

Reply:
How does a target modify a disease? Agreed, causing is not better - how about "disease-associated target"?

Reply:
The idea is that binding a drug to that target modifies (ideally cures) the disease. But disease-associated target works perfectly fine, can you change it while being on it?